### DISCUSSION TITLE
Best set up for MovieLens1M

### DISCUSSION THREAD

**Discussion by SavvinaDaniil** [lenskit/lkpy#330] [General]
Hi! I have a general question, not exactly LKPY related. 
Has there been a study (e.g., in a paper, or some notebook) on what is the best LKPY hyperparameter setting for UserKNN and ItemKNN trained on MovieLens1M and MovieLens100K? Thanks!

**Comment by mdekstrand** [comment]
For those two algorithms, the original LensKit for Java paper has insight on good values: https://md.ekstrandom.net/pubs/lenskit

One of the backlog tasks is to prepare documentation for good parmeters for the different algorithms, but that has not yet been done.

**Reply by SavvinaDaniil** [reply]
Thank you, that's very helpful.
